rbd/rbd_test.go

```diff
@@ -104,6 +104,42 @@ func TestGetImageNames(t *testing.T) {
 	conn.Shutdown()
 }
 
+func TestImageResize(t *testing.T) {
+	conn, _ := rados.NewConn()
+	conn.ReadDefaultConfigFile()
+	conn.Connect()
+
+	poolname := GetUUID()
+	err := conn.MakePool(poolname)
+	assert.NoError(t, err)
+
+	ioctx, err := conn.OpenIOContext(poolname)
+	require.NoError(t, err)
+
+	name := GetUUID()
+	image, err := rbd.Create(ioctx, name, 1<<22, 22)
+	assert.NoError(t, err)
+
+	err = image.Resize((1 << 22) * 2)
+	assert.Equal(t, err, rbd.RbdErrorImageNotOpen)
+
+	err = image.Open()
+	assert.NoError(t, err)
+
+	err = image.Resize((1 << 22) * 2)
+	assert.NoError(t, err)
```

Comment:
After the resize is done can we fetch & assert anything about the new size?

Reply:
Sizes are rounded up to megabytes, so the (standard) requested size does not match (allocated image is larger). I'll use MB's for testing `Image.Resize()`, but in case it starts failing we may need to check for `>= requestedSize`.